source/get-started/virtual-machine-install/hyper-v.rst

```diff
@@ -15,41 +15,40 @@ Overview
 ********
 
 Hyper-V is a type 1 bare-metal hypervisor that runs directly on system
-hardware.
+hardware. It is available for Windows server and client operating systems.
 
-Hyper-V is available both on server and client Windows operating systems as
-`Hyper-V on Windows Server`_ and `Hyper-V on Windows 10`_.
+|CL| provides a virtual disk image for Hyper-V, which also includes
+a :ref:`Hyper-V specific kernel <compatible-kernels>` and drivers.
 
 Prerequisites
 *************
 
-* Enable virtualization, such as `Intel® Virtualization Technology`_
-  (Intel® VT) and `Intel® Virtualization Technology for Directed I/O`_ (Intel®
-  VT-d), on the host system from EFI/BIOS.
+* Enable virtualization on the host system from EFI/BIOS, such as:
 
-* Hyper-V installed on a capable Windows operating system. Refer to the
-  Microsoft documentation on `Install the Hyper-V role on Windows Server`_ or
-  `Install Hyper-V on Windows 10`_.
+  * `Intel® Virtualization Technology`_ (Intel® VT)
+  * `Intel® Virtualization Technology for Directed I/O`_ (Intel® VT-d)
 
-* Configure appropriate virtual networking in Hyper-V. Refer to the Microsoft
-  documentation on `Create a virtual network on Window Server`_ or `Create a
-  virtual network on Window 10`_.
+* Install Hyper-V on the appropriate Windows operating system:
+
+  * `Install the Hyper-V role on Windows Server`_
+  * `Install Hyper-V on Windows 10`_
+
+* Configure the appropriate virtual networking in Hyper-V:
+
+  * `Create a virtual network on Windows Server`_
```

Comment:
@puneetse  This link points to a Microsoft doc called "Create a virtual switch..." should we change "network" to "switch" here?

Reply:
No, even though the Microsoft document has different contet/headings between the two documents, the targeted content is the same. For our purposes and readers the name for the two links should be the same to avoid confusion.